com.sap.cloud.lm.sl.common/src/test/java/com/sap/cloud/lm/sl/common/model/json/PropertiesAdapterFactoryTest.java

```diff
@@ -73,7 +73,7 @@ private void assertTestProperties(Map<String, Object> actualProperties) {
 
     private static class Foo {
 
-        @JsonAdapter(PropertiesAdapterFactory.class)
+        @JsonAdapter(NumberAdapterFactory.class)
```

Comment:
These should no longer be necessary, right?

Reply:
Actually, they are needed because adapter can be used only on Maps at first level not on objects containing maps.